lib/express/server.js

```diff
@@ -164,16 +182,13 @@ function configureServer (app, routeConfiguringFunction, allowCors = true, baseP
   // set up start logging (which depends on bodyParser doing its thing)
   app.use(startLogFormatter);
 
-  routeConfiguringFunction(app, basePath);
+  addRoutes(app, basePath, plugins.map((p) => p.newMethodMap));
```

Comment:
should we check first if newMethodMap is a plain object?

Reply:
👍